src/features/notifications/components/HeaderBanner.tsx

```diff
@@ -47,6 +49,15 @@ export const HeaderBanner: React.FC<{ bannerContent?: BannerContent }> = ({ bann
           </a>
         )}
       </p>
+      <button
+        className={headerbannerCustom.dismiss}
+        onClick={() => {
+          localStorage.setItem("headerBannerDismissed", "true")
```

Comment:
if possible save the ID of the current banner (if it exist) or just the URL or the title..

The idea is that when you dismiss a notification, should not show until the NEXT notification is shown. Currently it is handling notification as just one lot

The change also need to be implemented when reading the localStorage